cmd/minishift/cmd/openshift/service_list.go

```diff
@@ -53,6 +74,6 @@ var serviceListCmd = &cobra.Command{
 }
 
 func init() {
-	serviceListCmd.Flags().StringVarP(&serviceListNamespace, "namespace", "n", "default", "The namespace of the services.")
+	serviceListCmd.Flags().StringVarP(&serviceListNamespace, "namespace", "n", "", "The namespace of the services.")
```

Comment:
@praveenkumar @hferentschik IMO we should keep the current name  space(i.e. the name space user is using currently) space to list services. Refer #927